### ISSUE TITLE
Localization possibilities

### ISSUE THREAD

**Issue by marapper** [microsoft/accessibility-insights-web#1701]
## Is your feature request related to a problem? Please describe.

Here in Russia we have also WCAG-like accessibility laws and accessibility' movement growing support by developers and companies. It would be great if non-English speakers like QA and product owners could use this tool in theirs everyday work.

## Describe the desired outcome

Any of i18n libaries could help. But there is much more organisational issues and translations/reviews management.

Most of text components contains complex markdown with component inclusions so there should be some special markdown or structures like Link in `./src/content/link`.

To minimize translations synchronization issues language packs also can be extracted to separated repositories or separated from main build/releases and download-on-demand.

## Additional context

A11y insights is a fantastic tool! And different language translations could spread it around in the world.

**Comment by adarshrema** [comment]
@marapper, thank you for using Accessibility Insights for Web and this feedback. We triaged this and our current priorities do not allow us to focus on localization for now. We really appreciate your continued use of our tool and your interest in helping with wider adoption & usage.

**Comment by muan** [comment]
Any chance this can be reconsidered? Accessibility insights is a wonderful tool, and it'd be great if it could be available in different languages.
